src/commands/auth/login.ts

```diff
@@ -110,7 +110,7 @@ export default class Login extends Command {
 
     let loginData: LoginRecord | undefined
     if (refreshToken) {
-      loginData = { refreshToken }
+      loginData = { refreshToken, expires: Date.now() / 1000 + 60 }
```

Comment:
Is it configurable value for keycloak?

Reply:
I suppose yes, but this token is needed to do only one query right now